Makefile

```diff
@@ -29,7 +30,22 @@ pylint:
 flake8:
 	flake8 .
 
-test: lint unit-test
+test: lint unit-test tear-down-integration integration-test
 
 unit-test:
-	pytest
+	pytest --ignore=assisted-events-scrape/tests/integration
+
+integration-test: build-image
+	kind create cluster --name assisted-events-scrape || true
+	kind --name assisted-events-scrape export kubeconfig
+	kind load docker-image --name assisted-events-scrape $(ASSISTED_EVENTS_SCRAPE_IMAGE)
+	kubectl apply -f dev-manifests
+	./tools/generate_fixtures | kubectl delete -f - || true
+	./tools/generate_fixtures | kubectl create -f - || true
```

Comment:
``kubectl apply`` not good here?
be aware that even with ``delete``+``create``, you might still see the previous versions of the relevant objects in the next lines. so if relying on it somewhere, we might need to handle it to make sure there will be no race-conditions